ui/app/templates/components/radio-card.hbs

```diff
@@ -3,33 +3,44 @@
   class="radio-card {{if (eq @value @groupValue) 'is-selected'}} {{if @disabled 'is-disabled'}}"
   ...attributes
 >
-  {{#if (has-block)}}
-    {{yield}}
-  {{else}}
-    <div class="radio-card-row">
-      <div>
-        <Icon @name={{@icon}} @size="24" class="has-text-grey-light" />
+  <ToolTip @verticalPosition="above" @horizontalPosition="center" as |T|>
```

Comment:
`<ToolTip>` is inside `<label>` - so it wouldn't interfere with the `<div class="is-flex-row">` that wraps both `<RadioCard>` components